mobile/e2e-tests/scripts/live_word_monitor.py

```
@@ -203,6 +247,89 @@ class PreparedRow:
+
+
+def sanitize_device_id(device_id: str) -> str:
+    normalized = re.sub(r"[^a-zA-Z0-9._-]+", "_", device_id).strip("._-")
+    return normalized or "device"
+
+
+def get_device_display_name(device_id: str) -> str:
+    if device_id == "default":
+        return "Default Device"
+    try:
+        manufacturer_result = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)